pylot/component_creator.py

```diff
@@ -597,7 +605,7 @@ def add_control(pose_stream,
     return control_stream
 
 
-def add_evaluation(vehicle_id_stream, pose_stream, imu_stream):
+def add_evaluation(vehicle_id_stream: Stream[int], pose_stream: Optional[Stream[pylot.utils.Pose]] = None, imu_stream: Optional[Stream[IMUSetup]] = None) -> Stream:
```

Comment:
Return type is `None`. None of the arguments should be optional. `imu_stream` is `Stream[IMUMessageTuple]`.